src/Tween.js

```diff
@@ -7,906 +7,923 @@
  * Thank you all, you're awesome!
  */
 
+// UMD (Universal Module Definition)
+(function (root, factory) {
 
-var _Group = function () {
-	this._tweens = {};
-	this._tweensAddedDuringUpdate = {};
-};
+	if (typeof define === 'function' && define.amd) {
 
-_Group.prototype = {
-	getAll: function () {
+		// AMD
+		define([], function () {
+			return factory();
+		});
 
-		return Object.keys(this._tweens).map(function (tweenId) {
-			return this._tweens[tweenId];
-		}.bind(this));
+	} else if (typeof module !== 'undefined' && typeof exports === 'object') {
 
-	},
+		// Node.js
+		module.exports = factory();
```

Comment:
Note that `module.exports` is a Node.js thing; in the CommonJS Modules/1.1 spec, it's named `exports`.  Many modules are both Node.js and CommonJS modules by setting `exports = module.exports = ...`